public/css/login-registration.css

```diff
@@ -17,17 +26,21 @@
 
 /* GOOGLE LOGIN */
 hr {
-	border-top-color: #2b2b2b;
+	border-top-color: var(--black);
 	width: 70%;
+
+	margin-bottom: 12px;
 }
 
 #google {
 	height: 40px;
 }
 
 .warning {
-	margin-top: 5px;
-	font-size: 12px;
+	color: var(--black);
+	/* margin-top: 5px; */
```

Comment:
Can you delete this comment just to keep everything clean?

Reply:
Done! I checked on VS Code and it looks like class="fullscreen" is only for the login and registration, so we should be good on that. I removed the commented code too.